UnitedIncome/components
# [APP-4497] Percy Integration Fixes

### Description
> Please provide a brief description of your changes below.

Makes the required changes for Percy. With this change an auto accepted baseline will be accepted once the PR merges into `master`, it will still run on a PR from dev to master, but the baseline will be up-to-date and accurate preventing every snapshot from being flagfed.

Additionally I've added some skip logic into the GitHub action that will prevent commits with `skip ci` from being executed, preventing the auto bump functionality from running the snapshot twice. I've also skipped the `Portal` story from Percy as it always flags a single pixel diff due to it having a YouTube video in it.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* This will need to be tested with deployments. I've done some initial commits and everything appears to be functioning correctly.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4497)
